### ISSUE TITLE
Updates require a reload and don't finish while settings page is up

### ISSUE THREAD

**Issue by Osmose** [Osmose/advanced-open-file#87]
When updating to a new version, the package appears to not finish updating as long as the settings page is up; the progress animation on the update button loops forever until you leave and return. I imagine this is related to the fact that you need to reload the editor to see the effects of an update.

**Comment by Osmose** [comment]
Not entirely sure that the fix above worked or not, I can't really test it without doing a release, I think. We'll see!

**Comment by Osmose** [comment]
The fix didn't work for me with the 0.14.2 update. :(

**Comment by eboracus** [comment]
Yes, just to note, I'm definitely seeing exactly this still on the latest version. I didn't realise it was package specific though, as I'm pretty sure I've been seeing similar hanging on other packages too. Could it be all that require the editor reloading for an update to take effect? :s

Atom process CPU usage stays at a high activity level for me throughout too. Even though a CPU profile I took seems to suggest the actual update has finished just fine in a reasonable time as there's no processes recorded ongoing in the profile at the same time. Any relation here to https://github.com/atom/settings-view/issues/746?
